app/assets/stylesheets/application.css.scss

```diff
@@ -15,8 +15,32 @@
  *= require foundation_and_overrides
  */
 
+$white: #ffffff;
+$black: #111111;
+
+.logo {
+  background-color: #d9b82d;
```

Comment:
Color literals like `#d9b82d` should only be used in variable declarations; they should be referred to via variable everywhere else.